File: Data Structures & Algorithms/duplicate-integer/submission-0.go
func hasDuplicate(nums []int) bool {
    var m map[int]bool
    m = make(map[int]bool)

    for _, element := range nums {
        
        if m[element] {
            return true
        } else {
            m[element] = true
        }
    }

    return false
}

File: Data Structures & Algorithms/maximum-subarray/submission-0.cs
public class Solution {
    int max = -1000;
    
    public int MaxSubArray(int[] nums) {
        int total = 0;

        if (nums.Length > 1) {
            int[] numsCopy = new int[nums.Length - 1];

            Array.Copy(nums, 1, numsCopy, 0, nums.Length - 1);
            MaxSubArray(numsCopy);

            Array.Copy(nums, 0, numsCopy, 0, nums.Length - 1);
            MaxSubArray(numsCopy);
        }

        foreach (int element in nums)
        {
            total += element;
        }

        if (total > max) {
            max = total;
        }

        return max;
    }
}
